ui/src/app/views/cohort-builder/search/cohort-builder/cohort-builder.component.css

```diff
@@ -1 +1,20 @@
+.scrolling-groups {
+    height: 95%;
```

Comment:
I think this is partially contributing to the double scroll bar you can see in this screenshot:
![scrollbars](https://user-images.githubusercontent.com/1545444/30503867-e5dd9698-9a39-11e7-9a2b-0c3d60a2b1cb.png)
We should fix this sooner rather than later because it will get harder to track down as the codebase grows. I can definitely help. Let's pair up on this next week. No need to address in this PR since we'll need to touch a few points in the code to eliminate the problem.

Reply:
I have an idea of how to fix it, just haven't gotten around to doing it.